opensrp-app/build.gradle

```diff
@@ -128,11 +125,26 @@ android {
     }
 }
 
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
 tasks.withType(Test) {
     jacoco.includeNoLocationClasses = true
     testLogging {
         // to see the exceptions of failed tests in Travis-CI console.
-        exceptionFormat = 'full'
+        //exceptionFormat = 'full'
+        events TestLogEvent.FAILED,
+                TestLogEvent.PASSED,
+                TestLogEvent.SKIPPED,
+                TestLogEvent.STANDARD_OUT
+        exceptionFormat TestExceptionFormat.FULL
+        showExceptions true
+        showCauses true
+        showStackTraces true
+
+        error {
+            exceptionFormat = 'full'
+        }
```

Comment:
Is this needed for GitHub actions now that we switched from travis?

Reply:
Yes, I added this for Github actions specifically. I don't think there is an alternate option to enable the required logging